### PR TITLE
Use first positional argument as url to open

### LINKED ISSUE #187: Allow opening URLs without explicit command line switch
The issue:

    BROWSER='uzbl-browser -u'
    $BROWSER http://uzbl.org
    # => zsh: command not found: uzbl-browser -u

To avoid this, simply running `uzbl-browser http://uzbl.org` (without `-u/--uri`) should be possible.

### PR DESCRIPTION
**Pull request by keis**
Fixes #187

### CODE REVIEW

**Comment by keis** [comment]
@mathstuf Does this look good to you?

File: bin/uzbl-browser.in [hunk 1]
```diff
@@ -68,4 +68,4 @@ if [ -z "$UZBL_EVENT_SOCKET" ]; then
 	fi
 fi
 
-exec uzbl-core "$@" "${config_file:+--config "$config_file"}" --connect-socket "$UZBL_EVENT_SOCKET"
+exec uzbl-core "$@" ${config_file:+--config "$config_file"} --connect-socket "$UZBL_EVENT_SOCKET"
```
Comment (mathstuf) [inline bin/uzbl-browser.in]: Ooh, good catch.

**Comment by mathstuf** [comment]
+1

**Comment by mathstuf** [comment]
Oh, whoops. Documentation :) .

**Comment by keis** [comment]
Yes, ofc. Docs update now

**Pull request merged**